python/lsst/ctrl/mpexec/cli/cmd/commands.py

```diff
@@ -124,8 +126,33 @@ def build(ctx: click.Context, **kwargs: Any) -> None:
     _unhandledShow(show, "build")
 
 
+def start_coverage(coverage_packages: Tuple) -> Any:
+    print("Coverage turned ON!", file=sys.stderr)
+    if coverage_packages:
+        print("Coverage limited to: " + ",".join(coverage_packages), file=sys.stderr)
+
+    import coverage
+
+    coveragerc = os.path.join(pathlib.Path(__file__).parent.resolve(), "coveragerc")
+    cov = coverage.Coverage(
+        branch=True, concurrency="multiprocessing", config_file=coveragerc, source_pkgs=coverage_packages
```

Comment:
can't we say `config_file=False` here and then not need the coveragerc file at all? It looks like we are overriding the content anyhow.

Reply:
We need the coveragerc file because of the multiprocessing concurrency, otherwise it gives this error:

coverage.exceptions.ConfigError: multiprocessing requires a configuration file